### PR TITLE
  updated login with GitHub

### PR DESCRIPTION
**Pull request by jonathans199**

### CODE REVIEW

File: components/layout/Navigation/Tabs/StudentsAreaTab.tsx [hunk 1]
```diff
@@ -1,20 +1,20 @@
-import { useContext, useEffect, useState } from "react"
-import { NavDropdown } from "react-bootstrap"
-import { UserContext } from "../../../../pages/_app"
```
Comment (thedamian) [inline components/layout/Navigation/Tabs/StudentsAreaTab.tsx:3]: Do you have prettier or something? Can you add it to a .vsconfig or something so we all have it/

Comment (jonathans199) [reply components/layout/Navigation/Tabs/StudentsAreaTab.tsx:3]: @thedamian yes we do, in the repo, take a look, prettier is being applied upon every file staged in GIT. 
image

**Comment by andresmgomez** [comment]
I don't see it working locally. When I click on login it takes me to google auth, but it doesn't unlock the **Students Area**. Also the login modal was removed. I'm gonna check dev console to see if it stored the auth token for github user.

**Comment by thedamian** [comment]
@jonathans199  my comment was based on a lot of changes from " to ' and a lot of changes that I feel weren't you but instead prettier "fixing" how others work. You'll see a lot of that if you don't all use (force?) the same prettier and styling changes.

@andresmgomez If you're right it may be with what I mentioned earlier (and that's for you @jonathans199  ) I feel you're not keeping the login info or the navitation is reloading (real a href style) instead of just routing within react, and because localhost doesn't seem to be used, the moment you able to redirect to the right page it forgets that I was logged in.

Jonathan can you test that. It would seem that this should be part of the authentication task (in my oppinion)

**Review by brunomaria71 — APPROVED** [review]
looks good, merged

**Pull request merged**